stream_alert/athena_partition_refresh/main.py

```diff
@@ -29,25 +29,92 @@
 LOGGER.setLevel(LEVEL.upper())
 
 
+def _backoff_handler(details):
+    """Backoff logging handler for when polling occurs.
+
+    Args:
+        details (dict): Backoff context containing the number of tries,
+            target function currently executing, kwargs, args, value,
+            and wait time.
+    """
+    LOGGER.debug('[Backoff]: Trying again in %f seconds after %d tries calling %s',
+                 details['wait'],
+                 details['tries'],
+                 details['target'])
+
+def _success_handler(details):
+    """Backoff logging handler for when backoff succeeds.
+
+    Args:
+        details (dict): Backoff context containing the number of tries,
+            target function currently executing, kwargs, args, value,
+            and wait time.
+    """
+    LOGGER.debug('[Backoff]: Completed after %d tries calling %s',
+                 details['tries'],
+                 details['target'])
+
+def _load_config():
+    """Load the StreamAlert Athena configuration files
+
+    Returns:
+        [dict] Configuration settings by file, includes two keys:
```

Comment:
if we're going to switch to the `(dict)` style I'd suggest doing it everywhere in this file so it's not forgotten about in the future